bin/autojump

```diff
@@ -135,10 +135,13 @@ def add_path(data, path, weight=10):
     with resulting duplicate entries in the database than a single canonical
     path.
     """
-    path = unico(path).rstrip(os.sep)
```

Comment:
This eliminates the ability to handle filesystems with different encodings, particularly with Python 2.